libpod/pod.go

```diff
@@ -50,17 +50,22 @@ type PodConfig struct {
 // podState represents a pod's state
 type podState struct {
 	// CgroupPath is the path to the pod's CGroup
-	CgroupPath string
+	CgroupPath string `json:"cgroupPath"`
```

Comment:
podState is an odd name for a strut that only has a path in it....

Reply:
It will eventually have more - I think @haircommander is adding a container ID for the pause container in his PR.